src/hacbs/components/Modals/HacBSGettingStartedModal.tsx

```diff
@@ -0,0 +1,239 @@
+import * as React from 'react';
+import {
+  Button,
+  Modal,
+  ModalVariant,
+  Title,
+  Text,
+  TextVariants,
+  Alert,
+  GridItem,
+  Grid,
+} from '@patternfly/react-core';
+import classnames from 'classnames';
+import { LOCAL_STORAGE_KEY } from '../../../components/modal/GettingStartedModal';
+import { useLocalStorage } from '../../../hooks';
+import ExternalLink from '../../../shared/components/links/ExternalLink';
+
+import './HacBSGettingStartedModal.scss';
+
+type GettingStartedModalProps = {
+  imgClassName?: string;
+  localStorageKey: string;
+  title?: string;
+  imgSrc?: string;
+  imgAlt?: string;
+};
+
+type Level = {
+  header: {
+    title: string;
+    subText?: string;
+    image?: {
+      imgSrc: string;
+      imgAlt: string;
+      imgClass?: string;
+    };
+  };
+  content: {
+    visualization?: React.ReactElement;
+    sectionHeading?: string;
+    text: { key: number; jsx: React.ReactElement }[];
+  };
+  preFooter?: React.ReactElement;
+  footerButton: { title: string; cta: any };
+};
+
+export const HacBSGettingStartedModal: React.FC<GettingStartedModalProps> = ({
+  imgClassName,
+  localStorageKey,
+  imgSrc,
+  imgAlt,
+}) => {
+  const [storageKeys, setStorageKeys] =
+    useLocalStorage<{ [key: string]: boolean }>(LOCAL_STORAGE_KEY);
+  const keys = storageKeys && typeof storageKeys === 'object' ? storageKeys : {};
+  const isDismissed = keys[localStorageKey];
+  const [currentLevel, setCurrentLevel] = React.useState<number>(0);
+
+  const levels: Level[] = [
+    {
+      header: {
+        title: 'Welcome to App Factory',
+        image: { imgSrc, imgAlt, imgClass: imgClassName },
+      },
+      content: {
+        sectionHeading: 'One station to productization',
+        text: [
+          {
+            key: 1,
+            jsx: <>Complete automation, form commit to application release in one click</>,
+          },
+          { key: 2, jsx: <>Maximize your security with our off-the shelf CI/CD solutions</> },
+        ],
+      },
+      footerButton: { title: 'Next', cta: () => setCurrentLevel(1) },
+    },
+    {
+      header: {
+        title: 'How does it work',
+        subText: '1 of 2',
+      },
+      content: {
+        visualization: (
```

Comment:
These visualizations will likely just be an img. But I suppose we could also generate one with mock data.
cc @karthikjeeyar

Reply:
using images now